homeworks/iryna.gychka_irina-hychka/homework_5/task_1.js

```diff
@@ -0,0 +1,81 @@
+// Task 1.1 Converts a numeric array to array of sums
+const xArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const yArray = [0, 0, 0, 0, 0];
+const zArray = [10, -10, 10, -10, 10];
+
+function sumArray(n) {
+    return n.map((v, i) => {
+        let sum = 0;
+        for (let j = 0; j <= i; j++) {
+            sum += n[j];
+        }
+        return sum;
+    });
+}
```

Comment:
unfortunately, this is wrong... no need to use `for` loop inside map... this task is to understand closures

you have an array, you need to produce another array from this array... this is a typical `map()` operation

you have to have aside something like summarizer variable to keep track of current sum

=====

other tasks are ok, but in the first task you have to find more appropriate approach